go/vt/binlog/grpcbinlogplayer/player.go

```diff
@@ -41,7 +41,17 @@ type client struct {
 func (client *client) Dial(tablet *topodatapb.Tablet, connTimeout time.Duration) error {
 	addr := netutil.JoinHostPort(tablet.Hostname, tablet.PortMap["grpc"])
 	var err error
-	client.cc, err = grpcclient.Dial(addr, grpc.WithInsecure(), grpc.WithTimeout(connTimeout))
+
+	opts := []grpc.DialOption{
+		grpc.WithInsecure(),
+		grpc.WithTimeout(connTimeout),
+	}
+	opts, err = grpcclient.StaticAuthDialOption(opts, grpcclient.GetAuthStaticClientCreds())
```

Comment:
This boiler-plate should be avoided because we have to make this optional. Explained in grpcclient package.